src/test/resources/json/addfinaljudgmenttransferconfirmation_mutation_somedata.json

```diff
@@ -0,0 +1,3 @@
+{
+  "query": "mutation {addFinalJudgmentTransferConfirmation(addFinalJudgmentTransferConfirmationInput: {consignmentId: \"b42dccf0-549a-4204-bc9e-c6b69560b7a5\", legalCustodyTransferConfirmed: true}) {consignmentId, legalCustodyTransferConfirmed}}"
+}
```

Comment:
Could you put a line break at the end of the file please, keeps things consistent.